plugins/doc_fragments/auth.py

```diff
@@ -72,6 +73,13 @@ class ModuleDocFragment(object):
       jwt:
         description: The JSON Web Token (JWT) to use for JWT authentication to Vault.
         type: str
+      kubernetes_token:
+        description: The Kubernetes Token (JWT) to use for Kubernetes authentication to Vault.
+        type: str
+      kubernetes_token_path:
+        description: If no kubernetes_token is specified, will try to read the token from this path.
+        default: '/var/run/secrets/kubernetes.io/serviceaccount/token'
+        type: str
```

Comment:
When we're adding a new option, we can add `version_added` at the option level to mark when it was added, and that applies to env/ini/vars too (those have separate fields in case they were added to an option later).

I'm about to release 2.3.0 as there's some new stuff merged, so this will be slated for 2.4.0 for now. Not to worry though, I don't have a set release schedule for this collection, typically I do a release when there's something new.

```suggestion
      kubernetes_token:
        description: The Kubernetes Token (JWT) to use for Kubernetes authentication to Vault.
        type: str
        version_added: 2.4.0
      kubernetes_token_path:
        description: If no kubernetes_token is specified, will try to read the token from this path.
        default: '/var/run/secrets/kubernetes.io/serviceaccount/token'
        type: str
        version_added: 2.4.0
```